mcp/playbooks.go

```diff
@@ -27,6 +26,53 @@ import (
 	"github.com/flanksource/incident-commander/playbook"
 )
 
+// playbookToolNameToID maps tool names to playbook IDs
+var playbookToolNameToID = sync.Map{}
+
+const (
+	toolGetPlaybookRunSteps   = "get_playbook_run_steps"
+	toolGetFailedPlaybookRuns = "playbook_failed_runs"
+	toolGetRecentPlaybookRuns = "playbook_recent_runs"
+	toolPlaybooksListAll      = "playbooks_list_all"
+)
+
+func addPlaybooksAsTool(goctx gocontext.Context, srv *server.MCPServer, session server.ClientSession) error {
+	sessionID := session.SessionID()
```

Comment:
The test suite server is stateless, so you might have to modify that if you want to test these